libs/design-system/images/content/external-link.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M9.99978 5.82491C9.50273 5.82491 9.09978 6.22786 9.09978 6.72492C9.09978 7.22197 9.50273 7.62492 9.99978 7.62492L15.102 7.62492L6.41356 16.3134C6.06208 16.6648 6.06208 17.2347 6.41356 17.5862C6.76503 17.9376 7.33488 17.9376 7.68635 17.5862L16.5495 8.72301L16.5495 14.1746C16.5495 14.6717 16.9524 15.0746 17.4495 15.0746C17.9466 15.0746 18.3495 14.6717 18.3495 14.1746V6.72492C18.3495 6.22786 17.9466 5.82491 17.4495 5.82492L9.99978 5.82491Z" fill="currentCOlor"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9.99978 5.82491C9.50273 5.82491 9.09978 6.22786 9.09978 6.72492C9.09978 7.22197 9.50273 7.62492 9.99978 7.62492L15.102 7.62492L6.41356 16.3134C6.06208 16.6648 6.06208 17.2347 6.41356 17.5862C6.76503 17.9376 7.33488 17.9376 7.68635 17.5862L16.5495 8.72301L16.5495 14.1746C16.5495 14.6717 16.9524 15.0746 17.4495 15.0746C17.9466 15.0746 18.3495 14.6717 18.3495 14.1746V6.72492C18.3495 6.22786 17.9466 5.82491 17.4495 5.82492L9.99978 5.82491Z" />
```

Comment:
I think it was just a typo!

```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M9.99978 5.82491C9.50273 5.82491 9.09978 6.22786 9.09978 6.72492C9.09978 7.22197 9.50273 7.62492 9.99978 7.62492L15.102 7.62492L6.41356 16.3134C6.06208 16.6648 6.06208 17.2347 6.41356 17.5862C6.76503 17.9376 7.33488 17.9376 7.68635 17.5862L16.5495 8.72301L16.5495 14.1746C16.5495 14.6717 16.9524 15.0746 17.4495 15.0746C17.9466 15.0746 18.3495 14.6717 18.3495 14.1746V6.72492C18.3495 6.22786 17.9466 5.82491 17.4495 5.82492L9.99978 5.82491Z" fill="currentColor"/>
```

Reply:
ooohhh nice catch! I must've fat fingered it lol